pkgs/misc/emulators/duckstation/default.nix

```
@@ -0,0 +1,33 @@
+{ lib, mkDerivation, fetchFromGitHub, fetchpatch, cmake, pkgconfig, SDL2, qtbase
+, wrapQtAppsHook, qttools, ninja, git, gtk3, glib }:
+mkDerivation rec {
+  pname = "duckstation";
+  version = "unstable-2020-12-28";
```

The updated commit is from 2020-12-29 (please also change the commit message)

```suggestion
  version = "unstable-2020-12-29";
```